### ISSUE TITLE
funny geiger counter (punjabi) (halal) (100% virus free) (100% working)

### ISSUE THREAD

**Issue by ppgplayer** [studio-minus/ppg-bugs-and-requests#816]
**Feature description**
Just a regular ol geiger counter.

**Feature purpose**
I saw you were thinking of a radiation system and though this would be perfect for it.

Goes from 1 msv to "80+" S

**Additional context**
![image](https://user-images.githubusercontent.com/98086053/157829003-b433230c-e89e-489c-be37-c0e86e3db051.png)

**Comment by ppgplayer** [comment]
Mestiez changed my title :(

**Comment by SperoTeStupriMori** [comment]
that is already what is mestiez is gonna do.

**Comment by ppgplayer** [comment]
> that is already what is mestiez is gonna do.

This is a suggestion for a geiger counter specifically suggested for the radiation system which I knew was already being implemented

You clearly look through suggestions to be a white knight instead of reading it.

**Comment by ppgplayer** [comment]
> that is already what is mestiez is gonna do.

I know that you didnt read it because this was the seconds sentence of the suggestion 
![image](https://user-images.githubusercontent.com/98086053/158158035-60bf9feb-fce3-4f1b-b385-61e97d055816.png)

**Comment by SperoTeStupriMori** [comment]
> > that is already what is mestiez is gonna do.
> 
> This is a suggestion for a geiger counter specifically suggested for the radiation system which I knew was already being implemented
> 
> You clearly look through suggestions to be a white knight instead of reading it.

YEAH HE IS ALREADY GONNA ADD A GEIGER COUNTER, WASN'T IT OBVIOUS ENOUGH?

**Comment by ppgplayer** [comment]
> > > that is already what is mestiez is gonna do.
> > 
> > 
> > This is a suggestion for a geiger counter specifically suggested for the radiation system which I knew was already being implemented
> > You clearly look through suggestions to be a white knight instead of reading it.
> 
> YEAH HE IS ALREADY GONNA ADD A GEIGER COUNTER, WASN'T IT OBVIOUS ENOUGH?

tell me where he said that
